src/ansible_navigator/runner/base.py

```diff
@@ -20,7 +18,7 @@
 
 
 class Base:
-    """Base class"""
+    """Base class."""
```

Comment:
Probably need a better description of what this base class is for

Reply:
Added a bit more.. check it out.